frontend/src/App.tsx

```diff
@@ -48,6 +54,14 @@ export default class App extends React.Component<AppProps, LanguageContextValue>
                             <Route exact path='/'>
                                 <Redirect to="/home" />
                             </Route>
+
+                            <Route path='/yogori' component={YogiriPage}/>
```

Comment:
Should be `yogiri`

Reply:
If I change this to only "yogori" and in the headerNav too, then the link won't work anymore. Do you mean that the full path should be "/home/yogori"?

Reply:
I mean, the spelling for the path should be `/yogiri` here and in headerNav. Right now it's spelled with an `o` instead of an `i`.

Reply:
Ah did not notice the typos..